Client/src/Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -120,21 +124,21 @@ private Session CreateSession(IEnumerable<string> partitionIds)
   {
     using var _ = Logger.LogFunction();
     Logger.LogDebug("Creating Session... ");
-    var createSessionRequest = new CreateSessionRequest
-                               {
-                                 DefaultTaskOption = TaskOptions,
-                                 PartitionIds =
-                                 {
-                                   partitionIds,
-                                 },
-                               };
-    var session = ChannelPool.WithChannel(channel => new Api.gRPC.V1.Submitter.Submitter.SubmitterClient(channel).CreateSession(createSessionRequest));
+    var client = ChannelPool.WithChannel(channel => new Sessions.SessionsClient(channel));
+    var createSessionReply = client.CreateSession(new CreateSessionRequest
```

Comment:
Either you use the channel within the function `WithChannel`, or you create a local channel with get:
```c#
using var channel = ChannelPool.GetChannel();
```